docs/getting_started.rst

```diff
@@ -59,227 +56,207 @@ view an inventory of objects contained in the Document, simply
+Constructors can take additional arguments, depending on whether the SBOL class has required attributes. Attributes are required if the specification says they are. In a UML diagram, required attributes are indicated as properties with a cardinality of 1 or more. For example, a ``Component`` (see the UML diagram above) has only one required attribute, ``types``, which specifies one or more molecular types for a component.  Required attributes MUST be specified when calling a constructor. 
 
-   >>> setHomespace('')
-   >>> Config.setOption(ConfigOptions.SBOL_COMPLIANT_URIS, False)
-   >>> Config.setOption(ConfigOptions.SBOL_TYPED_URIS, False)
-   >>> crispr_template = ModuleDefinition('http://sbols.org/CRISPR_Example/CRISPR_Template')
-   >>> print(crispr_template)
-   http://sbols.org/CRISPR_Example/CRISPR_Template
-
-.. end
-
-The second set of configuration options demonstrates use of a default namespace for constructing URIs. The advantage of this approach is simply that it reduces repetitive typing. Instead of typing the full namespace for a URI every time an object is created, the user simply specifies the local identifier. The local identifier is appended to the namespace. This is a handy shortcut especially when working interactively in the Python interpreter.
+The following code creates a protein component (``types`` set to ``SBO_PROTEIN``). 
 
 .. code:: python
 
-   >>> setHomespace('http://sbols.org/CRISPR_Example/')
-   >>> Config.setOption(ConfigOptions.SBOL_COMPLIANT_URIS, False)
-   >>> Config.setOption(ConfigOptions.SBOL_TYPED_URIS, False)
-   >>> crispr_template = ModuleDefinition('CRISPR_Template')
-   >>> print(crispr_template)
-   http://sbols.org/CRISPR_Example/CRISPR_Template
+    >>> cas9 = sbol3.Component('Cas9', sbol3.SBO_PROTEIN)
 
 .. end
 
-The third set of configuration options demonstrates SBOL-compliant mode. In this example, a version number is appended to the end of the URI. Additionally, when operating in SBOL-compliant mode, the URIs of child objects are algorithmically constructed according to automated rules (not shown here).
+The following code creates a DNA component (``types`` set to ``SBO_DNA``). 
 
 .. code:: python
 
-   >>> setHomespace('http://sbols.org/CRISPR_Example/')
-   >>> Config.setOption(ConfigOptions.SBOL_COMPLIANT_URIS, True)
-   >>> Config.setOption(ConfigOptions.SBOL_TYPED_URIS, False)
-   >>> crispr_template = ModuleDefinition('CRISPR_Template')
-   >>> print(crispr_template)
-   http://sbols.org/CRISPR_Example/CRISPR_Template/1.0.0
+    >>> target_promoter = sbol3.Component('target_promoter', sbol3.SBO_DNA)
```

Comment:
I'd suggest that we also add the role here for a promoter.

Reply:
```suggestion
    >>> target_promoter = sbol3.Component('target_promoter', sbol3.SBO_DNA, roles=[sbol3.SO_PROMOTER])
```